manifests/install.pp

```diff
@@ -41,7 +41,9 @@
   package { $uchiwa::package_name:
     ensure  => $uchiwa::version,
     require => $repo_require,
-    notify  => Service['uchiwa'],
   }
 
+  if $::uchiwa::manage_services {
```

Comment:
Can you remove this change? It is too specific and operating directly on the package.

It belongs in the init class

Class['Uchiwa::install'] ~> Class['uchiwa::service']

And then let the manage_services gate inside the service class handle whether or not to do anything.